editor/export_template_manager.cpp

```diff
@@ -236,6 +236,7 @@ bool ExportTemplateManager::_install_from_file(const String &p_file, bool p_use_
 		}
 
 		ret = unzGoToNextFile(pkg);
+		ERR_BREAK(ret != UNZ_EOF && ret != UNZ_OK);
```

Comment:
I'd prefer the unzip changes moved to their own PR, with more explanations on what prompted the change and what it solves.